acceptance/current/.acceptance/acceptance-cookbook/.chef/config.rb

```diff
@@ -0,0 +1 @@
+cookbook_path File.join(File.dirname(__FILE__), '..', '..')
```

Comment:
Yeah the [chef-acceptance embedded `chef-client` run](https://github.com/chef/chef-acceptance/blob/master/lib/chef-acceptance/test_suite.rb#L45-L73) should render a temporary `config.rb` on the fly like `delivery-cli` does:
https://github.com/chef/delivery-cli/blob/7acaafed2291e644eed97fdda07ac634ad5f8d6f/src/delivery/job/workspace.rs#L51-L68